documentation/developer_documentation/CLI.md

```diff
@@ -25,6 +25,15 @@ The syntax for the `FULLNODE_API_INFO` variable is as follows:
 This will use IPv4, TCP, and HTTP when communicating with the RPC API. The admin token can be found when starting
 the Forest daemon. This will be needed to create tokens with certain permissions such as read, write, sign, or admin.
 
+## Token flag
+
+For nodes running on default port and when you are interacting locally, the admin token can also be set using `--token` flag:
+```
+forest-cli --token eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJBbGxvdyI6WyJyZWFkIiwid3JpdGUiLCJzaWduIiwiYWRtaW4iXSwiZXhwIjoxNjczMjEwMTkzfQ.xxhmqtG9O3XNTIrOEB2_TWnVkq0JkqzRdw63BdosV0c <subcommand>
+```
+
+Note that the Admin token is printed in the logs when starting Forest node.
```

Comment:
you could mention that it can be obtained with `--save-token <token_path>` as well https://github.com/ChainSafe/forest/pull/2577